packages/mobile/src/components/summary-table/SummaryTable.tsx

```diff
@@ -24,9 +25,6 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   lastRow: {
     borderBottomWidth: 0
   },
-  title: {
-    letterSpacing: 1
```

Comment:
how come?

Reply:
idk if they changed designs on us or what, but this is no longer in the figma.
but good callout cuz the title for purchase summary table also changed to "Transaction Summary"??

Reply:
wat. maybe reach out 👺

Reply:
seems like it was already updated on web but not mobile.